web-ng/src/app/components/select-dialog/select-dialog.html

```diff
@@ -0,0 +1,32 @@
+<!--
```

Comment:
This dialog is specific to selecting features - maybe better to same "select-feature-dialog`?